docs/tutorial/project-initialization.md

```diff
@@ -0,0 +1,83 @@
+---
+id: project-initialization
+title: Initialization of your Tartiflette recipes manager GraphQL API.
+sidebar_label: 2. Project initialization
+---
+
+We are going to create a Tartiflette recipes manager GraphQL, an exciting application which will allow you to manage efficiency all your recipes.
+
+Here is a list of some features we're going to implement:
+
+* List recipes _(`Query` operation)_
+* Add new recipe _(`Mutation` operation)_
+* Launch and wait the end of the cooking process _(`Subscription` operation)_ 
+* Add a limitation on creating recipe _(How to implement `directives`)_
+* Limit the administration part to the administrator only _(Secure nodes with `directives`)_
+* Do not expose all your GraphQL API for everyone, limit the administration features, `Mutation`, to administrators.
```

Comment:
Same as above, no ?

Reply:
Not really, this following will focus on protecting a field.
```
Limit the administration part to the administrator only _(Secure nodes with `directives`)_
```

this following will focus on do not expose the management features to everyone (dynamic introspection)
```
Do not expose all your GraphQL API for everyone, limit the administration features, `Mutation`, to administrators.
```

According to you, how should we name it to be as clear as possible?